benchmark/CMakeLists.txt

```diff
@@ -121,8 +121,7 @@ set(LIB_DIRS ${LIBHWLOC_LIBRARY_DIRS})
 if(UMF_BUILD_LIBUMF_POOL_DISJOINT)
     set(LIBS_OPTIONAL ${LIBS_OPTIONAL} disjoint_pool)
 endif()
-if(UMF_BUILD_LIBUMF_POOL_JEMALLOC)
-    set(LIBS_OPTIONAL ${LIBS_OPTIONAL} jemalloc_pool ${JEMALLOC_LIBRARIES})
+if(UMF_POOL_JEMALLOC_ENABLED)
     set(LIB_DIRS ${LIB_DIRS} ${JEMALLOC_LIBRARY_DIRS})
```

Comment:
nit: so lib_dirs but no libs?

Reply:
Right, will remove in a separate PR

Reply:
Done: https://github.com/oneapi-src/unified-memory-framework/pull/964